repos/fdbt-site/src/pages/products/selectExports.tsx

```diff
@@ -474,12 +474,28 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
     const noc = getAndValidateNoc(ctx);
     const products = await getAllProductsByNoc(noc);
     const nonExpiredProducts = getNonExpiredProducts(products);
-
     const allPassengerTypes = await getAllPassengerTypesByNoc(noc);
 
+    const multiModalAttribute = getSessionAttribute(ctx.req, MULTI_MODAL_ATTRIBUTE);
+    const dataSource = multiModalAttribute ? 'tnds' : 'bods';
+    const allBodsServices = await getBodsOrTndsServicesByNoc(noc, dataSource);
+    const allServicesWithMatchingLineIds: Record<string, MyFaresService> = {};
+
+    const validProducts = nonExpiredProducts.filter((product) => {
```

Comment:
```suggestion
    const validProducts = nonExpiredProducts.filter((product) => {
    
      if (!product.lineId) {
            return true;
        }
```

I noticed some behaviour where any fare that wasn't a service-type one was getting filtered out and couldn't be selected for exporting.

I think to resolve this we need a check so that if lineId doesn't exist in the product table we count it as a valid product to display.